aioriak/client.py

```diff
@@ -338,6 +338,8 @@ def bucket(self, name, bucket_type='default'):
 
     async def get_keys(self, bucket):
         '''
+            coroutine
```

Comment:
As i see sphinxcontrib-asyncio does't work with sphinx-autobuild. I should copy-paste all docstring into documentation.